components/select/select-top-control.component.ts

```diff
@@ -114,6 +135,7 @@ export class NzSelectTopControlComponent implements OnChanges, OnInit {
   @Input() removeIcon: TemplateRef<NzSafeAny> | null = null;
   @Input() listOfTopItem: NzSelectItemInterface[] = [];
   @Input() tokenSeparators: string[] = [];
+  @Input() prefix: TemplateRef<NzSafeAny> | string | null = null;
```

Comment:
question: because it's a new development does'nt make it more sense to use an inputS based signal ?